src/modules/accommodations/entities/accommodations.entity.ts

```diff
@@ -14,7 +14,7 @@ import { AccommodationAmenity } from './accommodation-amenity.entity';
 import { AccommodationAddress } from './accommodation-address.entity';
 import { Account } from '#/modules/user/entities/account.entity';
 
-@Entity('accommodations')
+@Entity('accommodation')
```

Comment:
Why it was renamed?

Reply:
because it was different for migrations file and the entity file